packages/@react-aria/selection/src/useSelectableCollection.ts

```diff
@@ -111,88 +111,58 @@ export function useSelectableCollection(options: SelectableCollectionOptions): S
       return;
     }
 
+    const navigateToKey = (key: Key | undefined, childFocus?: FocusStrategy) => {
+      if (key != null) {
+        manager.setFocusedKey(key, childFocus);
+
+        if (e.shiftKey && manager.selectionMode === 'multiple') {
+          manager.extendSelection(key);
+        } else if (selectOnFocus) {
+          manager.replaceSelection(key);
+        }
```

Comment:
Just wanna make sure, what is the expected behavior here if `selectionMode` is "multiple" but the shift key isn't held down and `selectOnFocus` is true? It would select the newly focused option and deselect the rest of the options?

Reply:
Yes, exactly. 
I found WAI-ARIA Authoring Practices a little confusing about it, but I still think this behavior makes the most sense and is what the native select does too.